prime-router/src/main/kotlin/azure/SubmissionFunction.kt

```diff
@@ -31,12 +90,12 @@ class SubmissionFunction(
      * @return a list of submission history results.
      */
     @FunctionName("getSubmissions")
-    fun run(
+    fun submissions(
```

Comment:
why do we need this function in addition to `organizationSubmissions`?

Reply:
so that we can provide for two use cases:
1. I want to see `my` information (where my credentials tell the system who I am, e.g. I shouldn't have to know what the system calls me)
2. I want to see `their` information (where my credentials tell the system what I have permission to see)